src/components/formio/checkbox.tsx

```diff
@@ -23,32 +50,16 @@ const Checkbox: React.FC<CheckboxProps> = ({
   tooltip = '',
   description = '',
   onChange,
-}) => {
-  const {getFieldProps} = useFormikContext();
-  const {onChange: formikOnChange} = getFieldProps(name);
-  const {hasErrors} = useValidationErrors(name);
-  return (
-    <Component field={name} required={required} type="checkbox">
-      <div className="form-check checkbox">
-        <label className="form-check-label">
-          <Field
-            name={name}
-            as="input"
-            type="checkbox"
-            className={clsx('form-check-input', {'is-invalid': hasErrors})}
-            onChange={(e: React.ChangeEvent<any>) => {
-              formikOnChange(e);
-              onChange?.(e);
-            }}
-          />
-          <span>{label}</span>
-          {tooltip && ' '}
-          <Tooltip text={tooltip} />
-        </label>
-      </div>
-      {description && <Description text={description} />}
-    </Component>
-  );
-};
-
+}) => (
+  <Component field={name} required={required} type="checkbox">
+    <div className="form-check checkbox">
+      <label className="form-check-label">
+        <CheckboxInput name={name} label={label} onChange={onChange} />
+        {tooltip && ' '}
```

Comment:
I don't know much on this side, is there a better way of adding spacing between these two elements?

Reply:
Unfortunately not, React collapses all whitespace and if they're different components, it even removes the whitespace.